tests/tests_integration/test_api/test_workflows.py

```diff
@@ -352,7 +352,7 @@ def test_delete_multiple_non_existing(self, cognite_client: CogniteClient, new_w
 
     def test_retrieve_workflow(self, cognite_client: CogniteClient, persisted_workflow_list: WorkflowList) -> None:
         retrieved = cognite_client.workflows.retrieve(persisted_workflow_list[0].external_id)
-        assert retrieved == persisted_workflow_list[0]
+        assert retrieved.dump() == persisted_workflow_list[0].dump()
```

Comment:
This keep failing, but it is hard to see why so I dump the results to compare to dicts instead:
<img width="1318" height="519" alt="image" src="https://github.com/user-attachments/assets/e5965015-ba91-4a96-9061-4f00b5b1dc23" />